mm2src/coins/z_coin/z_rpc.rs

```diff
@@ -553,10 +508,11 @@ pub enum SyncStatus {
     },
 }
 
+#[allow(unused)]
 pub struct SaplingSyncLoopHandle {
```

Comment:
cargo clippy for both targets didnt find errors without #[allow(unused)]

Reply:
ty!